diff --git a/node_modules/react-native/Libraries/Components/TextInput/TextInput.js b/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
index 1e95eb6..ae814dd 100644
--- a/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
+++ b/node_modules/react-native/Libraries/Components/TextInput/TextInput.js
@@ -885,6 +885,7 @@ const TextInput: component(
 ) = function TextInput({
   ref: forwardedRef,
   allowFontScaling = true,
+  maxFontSizeMultiplier = 1.3,
   rejectResponderTermination = true,
   underlineColorAndroid = 'transparent',
   autoComplete,
@@ -904,6 +905,7 @@ const TextInput: component(
   return (
     <InternalTextInput
       allowFontScaling={allowFontScaling}
+      maxFontSizeMultiplier={maxFontSizeMultiplier}
       rejectResponderTermination={rejectResponderTermination}
       underlineColorAndroid={underlineColorAndroid}
       editable={readOnly !== undefined ? !readOnly : editable}
diff --git a/node_modules/react-native/Libraries/Text/Text.js b/node_modules/react-native/Libraries/Text/Text.js
index b08b8f3..c72a22e 100644
--- a/node_modules/react-native/Libraries/Text/Text.js
+++ b/node_modules/react-native/Libraries/Text/Text.js
@@ -44,6 +44,7 @@ const TextImpl: component(
   accessibilityLabel,
   accessibilityState,
   allowFontScaling,
+  maxFontSizeMultiplier = 1.3,
   'aria-busy': ariaBusy,
   'aria-checked': ariaChecked,
   'aria-disabled': ariaDisabled,
@@ -71,11 +72,12 @@ const TextImpl: component(
   selectionColor,
   suppressHighlighting,
   style,
-  ...restProps
+  ...restPropsWithoutMultiplier
 }: {
   ref?: React.RefSetter<TextForwardRef>,
   ...TextProps,
 }) => {
+  const restProps = {...restPropsWithoutMultiplier, maxFontSizeMultiplier};
   const _accessibilityLabel = ariaLabel ?? accessibilityLabel;
 
   let _accessibilityState: ?TextProps['accessibilityState'] =
